packages/field-base/test/input-slot-mixin.test.js

```diff
@@ -19,8 +19,8 @@ customElements.define(
       return html`<slot name="input"></slot>`;
     }
 
-    ready() {
-      super.ready();
+    constructor() {
```

Comment:
Looks like using `ready()` here was a mistake from the beginning and it doesn't match the way it is used in the real components, e.g `text-field`:

https://github.com/vaadin/web-components/blob/92ce54078db36901094c9d861a99b132c72204f6/packages/text-field/src/vaadin-text-field.js#L57-L60